code/Common/src/main/java/net/selenate/common/comms/req/SeReqElementSetAttribute.java

```diff
@@ -0,0 +1,70 @@
+package net.selenate.common.comms.req;
+
+import net.selenate.common.SelenateUtils;
+import net.selenate.common.comms.SeElementSelector;
+
+
+public class SeReqElementSetAttribute implements SeCommsReq {
+  private static final long serialVersionUID = 45749879L;
+
+  private final SeElementSelector selector;
+  private final String            attributeName;
```

Comment:
Is there a use case where you need to set multiple attributes?
If so, it would be better to provide a list of attribute name-value pairs here.